stagemonitor-web-servlet/src/main/java/org/stagemonitor/web/servlet/MonitoredHttpRequest.java

```diff
@@ -254,6 +270,10 @@ public void onFinish(SpanWrapper span, String operationName, long durationNanos)
 					logger.warn("Suppressed exception", e);
 				}
 			}
+			final Long gcTime = contextInfo.getRequestAttribute(GC_TIME_ATTRIBUTE);
+			if (gcTime != null) {
+				span.setTag("gc_time_ms", collectGcTimeMs() - gcTime);
```

Comment:
how to i know if the GC time was blocking (stop the world) or parallel? So maybe add multiple tags for each `GarbageCollectorMXBean#getName()`?